www/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
Easy to read but the indentation is a difficult to decipher at times. I think a general comment or two might be helpful.